src/Pkg/PParser.hs

```diff
@@ -2,7 +2,13 @@
 #if !(MIN_VERSION_base(4,8,0))
 {-# LANGUAGE OverlappingInstances #-}
 #endif
-
+--  ------------------------------------------------------------- [ PParser.hs ]
```

Comment:
We don't have this kind of header anywhere else in the source repository - why should it be introduced with this change?